net.tenshu.Terminator2.yml

```diff
@@ -0,0 +1,47 @@
+---
+id: net.tenshu.Terminator2
+base-version: main
+runtime: org.gnome.Sdk
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: terminator
+rename-desktop-file: terminator.desktop
+rename-appdata-file: terminator.appdata.xml
+rename-icon: terminator
+copy-icon: true
+
+finish-args:
+  - --share=ipc
+  - --socket=session-bus
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-run/gvfsd
+  - --filesystem=home
+  - --own-name=net.tenshu.Terminator2
+  - --talk-name=org.freedesktop.Flatpak
+
+modules:
+  - shared-modules/intltool/intltool-0.51.json
+  - modules/python3-packaging.yml
+  - modules/python3-tomli.yml
+  - modules/python3-setuptools_scm.yml
+  - modules/python3-pytest-runner.yml
+  - modules/python3-psutil.yml
+  - modules/python3-pycairo.yml
+  - modules/python3-pygobject.yml
+  - modules/python3-dbus-python.yml
+  - modules/python3-configobj.yml
+  - modules/keybinder.yml
+  - modules/vte.yml
```

Comment:
this is unmaintainable. Only auto-generated manifest should be separate. And for python usually you can have one for everything.

Reply:
Can you give a little bit more info on what is it exactly you're proposing?

Reply:
that 1 file per module is unmaintainable. If you use the flatpak-builder-tools for the python dependencies it can generate ONE file for all the python deps that would be included here. vte and keybinder definetly ain't.